Booktrix/Controllers/BookshelfView/UserBookViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  BookViewController.swift
+//  Booktrix
+//
+//  Created by Impresyjna on 04.06.2017.
+//  Copyright © 2017 Impresyjna. All rights reserved.
+//
+
+import UIKit
+
+protocol UserBookProtocol {
+    func scanFinished(book: Book)
+}
+
+final class UserBookViewController: UIViewController, UserBookProtocol {
+    @IBOutlet weak var contentContainer: UIView!
+    @IBOutlet weak var segmentedControl: UISegmentedControl!
+    @IBOutlet weak var titleTextField: UITextField!
+    @IBOutlet weak var authorTextField: UITextField!
+    @IBOutlet weak var coverImage: UIImageView!
+
+    var viewModel: UserBookViewModel!
+    
+    lazy var descriptionController : BookDescriptionViewController? = {
+        let vc = Wireframe.UserBookEditView().description()
+        vc.viewModel = UserBookDataViewModel(self.viewModel.form)
+        return vc
+    }()
```

Comment:
To pewnie byłoby za dużo roboty jak na czas jaki został do końca, ale takim super poprawnym podejściem byłoby zrobienie generycznego kontenera, który przyjmowałby jakiś data source gdzie można byłoby podawać te kontrolery (po prostu coś w stylu page view controller ale sterowane przez segmented). W ten sposób unikniesz zduplikowanej logiki